composer.json

```diff
@@ -30,6 +30,7 @@
     "require": {
         "php": "^7.4||^8.0",
         "ext-mbstring": "*",
+        "ext-json": "*",
```

Comment:
please use `composer require ext-json:*` to update the lockfile. 

and do also add this requirement to `packages/guides/composer.json` as that is the composer.json used on packagist.